### PR TITLE
137 use configmap

### PR DESCRIPTION
**Pull request by mohammad-sdsc**

### CODE REVIEW

File: helm-chart/renku-notebooks/templates/configmap.yaml [hunk 1]
```diff
@@ -23,3 +23,31 @@ metadata:
     heritage: {{ .Release.Service }}
 data:
   SENTRY_DSN: {{ .Values.jupyterhub.singleuser.sentryDsn }}
+---
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: hub-helper-scripts
```
Comment (rokroskar) [inline helm-chart/renku-notebooks/templates/configmap.yaml]: this is a very minor thing, but technically this is a helper script for jupyter notebooks not the hub... maybe the naming will be misleading?

Comment (mohammad-sdsc) [reply helm-chart/renku-notebooks/templates/configmap.yaml]: `notebook-helper-scripts` then?

Comment (rokroskar) [reply helm-chart/renku-notebooks/templates/configmap.yaml]: 👍

File: helm-chart/renku-notebooks/values.yaml [hunk 2]
```diff
@@ -143,7 +143,13 @@ jupyterhub:
       name: renku/singleuser
       tag: latest
     storage:
-      type: none
```
Comment (rokroskar) [inline helm-chart/renku-notebooks/values.yaml]: don't remove this!

File: helm-chart/renku-notebooks/values.yaml [hunk 3]
```diff
@@ -144,6 +144,15 @@ jupyterhub:
...
     storage:
       type: none
+      extraVolumes:
+        - name: notebook-helper-scripts-volume
+          configMap:
+            name: notebook-helper-scripts
+            defaultMode: 0755
+      extraVolumeMounts:
+        - name: notebook-helper-scripts-volume
+          mountPath: /usr/local/bin/pre-stop.sh
```
Comment (rokroskar) [inline helm-chart/renku-notebooks/values.yaml]: I guess this works for a single script, but what if we had multiple?

Comment (rokroskar) [reply helm-chart/renku-notebooks/values.yaml]: ah nevermind, it's fine 👍

**Pull request merged**